jina_cli/autocomplete.py

```diff
@@ -213,6 +213,7 @@
         'cloud login': ['--help'],
         'cloud logout': ['--help'],
         'cloud deploy': ['--help'],
+        'cloud normalize': ['--help'],
```

Comment:
What is this about?

Reply:
this is something about `Jina Dev Bot` pushing some commits behind the back